pages/recipes/supabase.mdx

```diff
@@ -1,24 +1,157 @@
 import { Callout } from "nextra/components";
 
-# Supabase
+# Overview
```

Comment:
I would still call the page "Supabase".

Reply:
Yes, agreed. Keep the # Supabase and add a ## Overview section imho.